circle.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
         github.com/golang/lint/golint \
         github.com/kisielk/errcheck \
         github.com/fatih/hclfmt
+    - pip install yapf flake8
```

Comment:
We need to make sure this is pinned to a particular version. Otherwise upstream yapf releases will break our build.